### PR TITLE
Fix: SSR issues and eslint warnings 

### LINKED ISSUE #1239: Next not rendering HTML - SEO Issue
Next SSR should be outputing HTML to the source, but right now it is fully rendering at the client. @jomarmontuya I need you to explore why this is happening. It may have something to do with the wrappers being used in app js like <page or <snackbar

please open a branch and play with removing things until you get it to out HTML

image

### PR DESCRIPTION
**Pull request by darwin808**
## Changes 

- Fix SSR issues
- handle undefined window
- replace `mui-markdown` with `markdown-to-jsx`
- remove console logs
- remove eslint warnings

### CODE REVIEW

File: src/components/cta/DemoForm.js [hunk 1]
```diff
@@ -38,34 +38,34 @@ const validationSchema = yup.object({
...
-    Lead_Source: 'Website',
-    // "Description": document.querySelector('#ac-description').value,
-    Role: roles,
-    // 'Project_Timeline' : document.querySelector('#ac-timeline').value,
-    Lead_Source_Detail: 'Demo Sign Up',
-    Business_Type: 'Direct',
-    Lead_Status: 'Not Contacted',
-  };
-};
+// const getLeadObjectZOHO = (obj, roles) => {
```
Comment (ardeay) [inline src/components/cta/DemoForm.js]: Why are leads commented out?

Comment (ardeay) [reply src/components/cta/DemoForm.js]: This object will need to come back

File: src/components/cta/TryFreeButton.js [hunk 2]
```diff
@@ -10,52 +9,52 @@ import Button from '@mui/material/Button';
...
-          strokeLinecap="round"
-          strokeLinejoin="round"
-          strokeWidth={2}
-          d="M20.354 15.354A9 9 0 018.646 3.646 9.003 9.003 0 0012 21a9.003 9.003 0 008.354-5.646z"
-        />
-      </svg>
-    ),
-  },
-];
+// const mock = [
```
Comment (ardeay) [inline src/components/cta/TryFreeButton.js]: delete

File: src/components/docs/docsLookup.js [hunk 3]
```diff
@@ -6,16 +6,16 @@ export async function docsLookup(ctx) {
   // setup default path
   let markdownFilePath = 'homepage.md';
   // some file structures have a README.md as the index, this will be use to check that
-  let markdownFilePathREADME = 'README.md';
+  // let markdownFilePathREADME = 'README.md';
```
Comment (ardeay) [inline src/components/docs/docsLookup.js]: bring back

File: src/components/wrappers/Page.js [hunk 4]
```diff
@@ -77,18 +54,15 @@ export default function Page({ children }) {
     });
   }, []);
 
-  const [themeMode, themeToggler, mountedComponent] = useDarkMode();
-
   useEffect(() => {
     AOS.refresh();
```
Comment (ardeay) [inline src/components/wrappers/Page.js:58]: @darwin808 wasnt this removed with the bug?

File: src/lib/protectedRouteGetServerSideProps.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+import { getCookies } from 'cookies-next';
```
Comment (ardeay) [inline src/lib/protectedRouteGetServerSideProps.js:1]: name space this  to `src/lib/accounts/protectedRouteGetServerSideProps.js`

**Review by ardeay — CHANGES_REQUESTED** [review]

File: src/components/wrappers/Page.js (hunk 4 above)
Comment (darwin808) [reply src/components/wrappers/Page.js:58]: image

@ardeay  this `AOS` is from an npm package and not from the cdn

**Comment by glespinosa** [comment]
done updating request changes above

**Comment by ardeay** [comment]
please make a separate PR from the branch into Stage @glespinosa

**Pull request closed without merging**